rule_repository_connector.bal

```diff
@@ -0,0 +1,208 @@
+import ballerinax/health.fhir.r4;
+
+# Implement this class to connect to the rule repository
+# This class should be able to fetch the rules from the repository
+# and make the decisions based on the rules
+isolated class RuleRepositoryConnector {
```

Comment:
The name should contain `mock` so that the devs would know this is just a sample impl. Also as per the earlier comment this should extend the class type.

Reply:
Ack